src/components/CardTable/CardTable.jsx

```diff
@@ -109,14 +109,16 @@ export default class CardTable extends Component {
   }
 
   get largestTableWidth () {
-    const tableNodes = Object.values(this.tableNodes)
+    const tableNodes = Object.values(this.tableNodes || {})
     const tableWidths = tableNodes.map(tableNode => tableNode.clientWidth)
 
     return Math.max(...tableWidths)
   }
 
   get parentWidth () {
-    return getElementContentWidth(this.containerNode.parentNode)
+    const ZERO = 0
+
+    return this.containerNode ? getElementContentWidth(this.containerNode.parentNode) : ZERO
```

Comment:
The issue was that `get parentWidth()` was being called when `this.containerNode` was null so it would throw an error (trying to get the parent node) before calling `getElementContentWidth` .

Although I'm struggling to reproduce this after fixing up the other bugs.